frontend/views/components/UsersSelector.vue

```diff
@@ -117,9 +115,6 @@ export default ({
   watch: {
     value () {
       this.$emit('change', this.value)
-    },
-    usernames () {
-      this.clear()
```

Comment:
Apparently `usernames` here was meant to be removed in [this previous commit](https://github.com/okTurtles/group-income/commit/f8cac6630b098bf693585db3fd23a60b5a9977a7) but went unnoticed, causing a run-time error. So removed it here.